platform/vs/docker-sonic-vs/database_config.json

```diff
@@ -11,6 +11,11 @@
             "port": 6380,
             "unix_socket_path": "/var/run/redis/redis_chassis.sock"
         }
+        ,"redis_bmp":{
```

Comment:
nit:  Can we put comma at the end of the previous section?

Reply:
done